Add status STALLED and change status of many proposed RFCs

swcurran opened the pull request:
We have many RFCs that are in the "Proposed" status that are very unlikely to be progressed any further. To clear them out so that people have a better view of the active work in Aries, a new status "STALLED" has been added, and the status of many proposed RFCs have been changed to either STALLED, DEMONSTRATED or ACCEPTED, as appropriate based on the work done on the RFCs.

STALLED differs from the RETIRED state in that STALLED is direct from PROPOSED, whereas RETIRED is from a more advanced state, where implementation work was done, evaluated and found to be no longer of use.  STALLED just means that the idea never got beyond the idea stage.

Updated statuses:

-  STALLED:   concepts/0029-message-trust-contexts/README.md
-  RETIRED:   concepts/0051-dkms/README.md
-  STALLED:   concepts/0207-credential-fraud-threat-model/README.md
-  STALLED:   concepts/0217-linkable-message-paths/README.md
-  STALLED:   concepts/0250-rich-schemas/README.md
-  STALLED:   concepts/0257-private-credential-issuance/README.md
-  STALLED:   concepts/0268-unified-didcomm-agent-deeplinking/README.md
-  STALLED:   concepts/0289-toip-stack/README.md
-  STALLED:   concepts/0420-rich-schemas-common/README.md
-  STALLED:   concepts/0478-coprotocols/README.md
-  STALLED:   concepts/0700-oob-through-redirect/README.md
-  STALLED:   features/0024-didcomm-over-xmpp/README.md
-  STALLED:   features/0030-sync-connection/README.md
-  STALLED:   features/0075-payment-decorators/README.md
-  STALLED:   features/0114-predefined-identities/README.md
-  STALLED:   features/0116-evidence-exchange/README.md
-  STALLED:   features/0193-coin-flip/README.md
-  STALLED:   features/0213-transfer-policy/README.md
-  STALLED:   features/0214-help-me-discover/README.md
-  STALLED:   features/0249-rich-schema-contexts/README.md
-  STALLED:   features/0281-rich-schemas/README.md
-  STALLED:   features/0309-didauthz/README.md
-  STALLED:   features/0327-crypto-service/README.md
-  STALLED:   features/0334-jwe-envelope/README.md
-  STALLED:   features/0335-http-over-didcomm/README.md
-  STALLED:   features/0347-proof-negotiation/README.md
-  STALLED:   features/0351-purpose-decorator/README.md
-  STALLED:   features/0418-rich-schema-encoding/README.md
-  STALLED:   features/0428-prepare-issue-rich-credential/README.md
-  STALLED:   features/0429-prepare-req-rich-pres/README.md
-  STALLED:   features/0445-rich-schema-mapping/README.md
-  STALLED:   features/0446-rich-schema-cred-def/README.md
-  STALLED:   features/0482-coprotocol-protocol/README.md
-  STALLED:   features/0641-linking-binary-objects-to-credentials/README.md
-  STALLED:   features/0693-credential-representation/README.md
-  DEMONSTRATED:   features/0755-oca-for-aries/README.md
-  DEMONSTRATED:   features/0756-oca-for-aries-style-guide/README.md
-  DEMONSTRATED:   features/0780-data-urls-images/README.md
-  ACCEPTED:   features/0794-did-rotate/README.md
-  DEMONSTRATED:   features/0804-didcomm-rpc/README.md
-  DEMONSTRATED:   features/0809-w3c-data-integrity-credential-attachment/README.md

TimoGlastra reviewed (approved):
Makes sense 👍

JamesKEbert commented:
I'm highly in favor of adding a `stalled` status. :+1: 

Here's some additional RFCs I'd propose for status changes:
ACCEPTED: features/0183-revocation-notification/README.md (it's included in AIPv2, so it should probably move to accepted--even though revocation notification v2 would be preferable for implementation)
ACCEPTED: features/0721-revocation-notification-v2/README.md
ACCEPTED: features/0685-pickup-v2/README.md
ADOPTED: features/0015-acks/README.md
ADOPTED: concepts/0017-attachments/README.md
ADOPTED: features/0035-report-problem/README.md
ADOPTED: features/0092-transport-return-route/README.md
ADOPTED: concepts/0094-cross-domain-messaging/README.md
ADOPTED: features/0211-route-coordination/README.md
ADOPTED: features/0360-use-did-key/README.md
ADOPTED: concepts/0441-present-proof-best-practices/README.md
ADOPTED: features/0557-discover-features-v2/README.md

swcurran commented:
At the [Aries Working Group meeting 2024-05-01](https://wiki.hyperledger.org/display/ARIES/2024-05-01+Aries+Working+Group+Call), we went through a list of all of the proposed changes in this PR, as listed in this [spreadsheet](https://docs.google.com/spreadsheets/d/1Dnp6qg4RZkdvC7G7izoPmHeWRk9S0mB7qKiwjWkaOJQ/edit?usp=sharing).

There was agreement and no objections to the following actions:

- For the RFCs marked "review" in the spreadsheet, revert the change to "STALLED", pending a further review. A new PR will be created should a change in status in those RFCs is called for.  The review list is:
  - 0268-unified-didcomm-agent-deeplinking
  - 0700-oob-through-redirect
  - 0213-transfer-policy
  - 0214-help-me-discover
  - 0347-proof-negotiation
  - 0641-linking-binary-objects-to-credentials
  - 0693-credential-representation
- For the RFCs proposed for status changes by @JamesKEbert in [this comment](https://github.com/hyperledger/aries-rfcs/pull/827#issuecomment-2066903319), we agreed to apply all of those changes.
- Once those updates to this PR have been made -- merge this PR.

As of the writing of this comment, all of those changes have been made.

@JamesKEbert -- could I get you to review the changes to make sure I didn't make a mistake in applying the spreadsheet changes?  Or anyone else.  The PR is approved so in theory COULD be applied, but I'd appreciate another pair of eyes on it.

Thanks!

JamesKEbert commented on index.md:
```diff
@@ -45,91 +45,93 @@
 * [0557: Discover Features Protocol v2.x](features/0557-discover-features-v2/README.md) (2021-04-15 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol) [`test-anomaly`](/tags.md#test-anomaly))
 * [0587: Encryption Envelope v2](features/0587-encryption-envelope-v2/README.md) (2021-04-15 &mdash; [`feature`](/tags.md#feature))
 * [0646: W3C Credential Exchange using BBS+ Signatures](features/0646-bbs-credentials/README.md) (2021-04-28 &mdash; [`feature`](/tags.md#feature))
+* [0755: Overlays Capture Architecture (OCA) For Aries](features/0755-oca-for-aries/README.md) (2024-03-02 &mdash; [`feature`](/tags.md#feature))
+* [0756: OCA for Aries Style Guide](features/0756-oca-for-aries-style-guide/README.md) (2023-01-05 &mdash; [`feature`](/tags.md#feature))
+* [0780: Use Data URLs for Images and More in Credential Attributes](features/0780-data-urls-images/README.md) (2024-03-02 &mdash; [`feature`](/tags.md#feature))
```
These were moved to `Demonstrated`, not `Accepted`.

JamesKEbert commented on index.md:
```diff
@@ -45,91 +45,93 @@
+* [0249: Aries Rich Schema Contexts](features/0249-rich-schema-contexts/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`rich-schemas`](/tags.md#rich-schemas))
+* [0250: Rich Schema Objects](concepts/0250-rich-schemas/README.md) (2024-04-03 &mdash; [`concept`](/tags.md#concept) [`rich-schemas`](/tags.md#rich-schemas))
+* [0257: Private Credential Issuance](concepts/0257-private-credential-issuance/README.md) (2024-04-03 &mdash; [`concept`](/tags.md#concept) [`protocol`](/tags.md#protocol))
+* [0268: Unified DIDCOMM Deeplinking](concepts/0268-unified-didcomm-agent-deeplinking/README.md) (2024-04-03 &mdash; [`concept`](/tags.md#concept))
```
Marked for review, should stay in proposed.

JamesKEbert commented on index.md:
```diff
@@ -45,91 +45,93 @@
+* [0116: Evidence Exchange Protocol 0.9](features/0116-evidence-exchange/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol))
+* [0193: Coin Flip Protocol 1.0 ](features/0193-coin-flip/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol))
+* [0207: Credential Fraud Threat Model](concepts/0207-credential-fraud-threat-model/README.md) (2024-04-03 &mdash; [`concept`](/tags.md#concept) [`credentials`](/tags.md#credentials))
+* [0213: Transfer Policy Protocol](features/0213-transfer-policy/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol))
+* [0214: "Help Me Discover" Protocol](features/0214-help-me-discover/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol))
```
Marked for review, should stay in proposed

JamesKEbert commented on index.md:
```diff
@@ -45,91 +45,93 @@
+* [0327: Crypto service Protocol 1.0](features/0327-crypto-service/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol))
+* [0334: JWE envelope 1.0](features/0334-jwe-envelope/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature))
+* [0335: HTTP Over DIDComm](features/0335-http-over-didcomm/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol))
+* [0347: Proof Negotiation](features/0347-proof-negotiation/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol))
```
Marked for review, should stay in proposed.

JamesKEbert commented on index.md:
```diff
@@ -45,91 +45,93 @@
+* [0446: Aries Rich Schema Credential Definition](features/0446-rich-schema-cred-def/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`rich-schemas`](/tags.md#rich-schemas))
+* [0478: Coprotocols](concepts/0478-coprotocols/README.md) (2024-04-03 &mdash; [`concept`](/tags.md#concept) [`protocol`](/tags.md#protocol))
+* [0482: Coprotocol Protocol 0.5](features/0482-coprotocol-protocol/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol))
+* [0641: Linking binary objects to credentials using hash based references](features/0641-linking-binary-objects-to-credentials/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature) [`credentials`](/tags.md#credentials))
+* [0693: Cross-Platform Credential Representation](features/0693-credential-representation/README.md) (2024-04-03 &mdash; [`feature`](/tags.md#feature))
+* [0700: Out-of-Band through redirect](concepts/0700-oob-through-redirect/README.md) (2024-04-03 &mdash; [`concept`](/tags.md#concept) [`decorator`](/tags.md#decorator))
```
Marked for review, should stay in proposed.

JamesKEbert commented on index.md:
```diff
@@ -45,91 +45,93 @@
+* [0780: Use Data URLs for Images and More in Credential Attributes](features/0780-data-urls-images/README.md) (2024-03-02 &mdash; [`feature`](/tags.md#feature))
 * [0793: Unqualified DID Transition](features/0793-unqualfied-dids-transition/README.md) (2023-07-11, [12 impls](features/0793-unqualfied-dids-transition/README.md#implementations) &mdash; [`feature`](/tags.md#feature) [`community-update`](/tags.md#community-update))
+* [0794: DID Rotate 1.0](features/0794-did-rotate/README.md) (2024-03-02 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol))
+* [0804: DIDComm Remote Procedure Call (DRPC)](features/0804-didcomm-rpc/README.md) (2024-03-02 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol))
+* [0809: W3C Verifiable Credential Data Integrity Attachment format for requesting and issuing credentials](features/0809-w3c-data-integrity-credential-attachment/README.md) (2024-01-10 &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol) [`credentials`](/tags.md#credentials) [`test-anomaly`](/tags.md#test-anomaly))
```
These were moved to Demonstrated, not Accepted.

JamesKEbert commented on index.md:
```diff
@@ -8,7 +8,7 @@
 * [0020: Message Types](concepts/0020-message-types/README.md) (2019-05-24, [8 impls](concepts/0020-message-types/README.md#implementations) &mdash; [`concept`](/tags.md#concept))
 * [0021: DIDComm Message Anatomy](concepts/0021-didcomm-message-anatomy/README.md) (2019-05-25 &mdash; [`concept`](/tags.md#concept) [`decorator`](/tags.md#decorator))
-* [0023: DID Exchange Protocol 1.0](features/0023-did-exchange/README.md) (2021-04-15, [1 impl](features/0023-did-exchange/README.md#implementations) &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol) [`test-anomaly`](/tags.md#test-anomaly))
+* [0023: DID Exchange v1](features/0023-did-exchange/README.md) (2021-04-15, [1 impl](features/0023-did-exchange/README.md#implementations) &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol) [`test-anomaly`](/tags.md#test-anomaly))
```
Missing 017, 094, 0441, 015, 035, 092, 0211, 0360, 0557

JamesKEbert commented on mkdocs.yml:
```diff
@@ -126,127 +126,128 @@ nav:
-    - 0207 Credential Fraud Threat Model: concepts/0207-credential-fraud-threat-model/README.md
     - 0212 Pickup Protocol: features/0212-pickup/README.md
-    - 0212 Pickup Protocol 2.0: features/0685-pickup-v2/README.md
-    - 0213 Transfer Policy Protocol: features/0213-transfer-policy/README.md
```
Marked for Review, should stay

JamesKEbert commented on features/0213-transfer-policy/README.md:
```diff
@@ -1,8 +1,8 @@
 # 0213: Transfer Policy Protocol
 - Authors: [Sam Curren](telegramsam@gmail.com)
-- Status: [PROPOSED](/README.md#proposed)
-- Since: 2019-09-03
-- Status Note: Initial version 
+- Status: [STALLED](/README.md#stalled)
+- Since: 2024-04-03
+- Status Note: No implementations have been created.
 - Start Date: 2019-09-03
 - Tags: [feature](/tags.md#feature), [protocol](/tags.md#protocol)
 
```
Marked for review, should stay proposed.

JamesKEbert reviewed (changes requested):
Looks good! Just a couple of minor catches. :+1:

swcurran commented:
Sorry about that @JamesKEbert — I ran the site generator script thinking that would be update the index as well. So, changed the script so now it does that, and included the updated index.md in this PR.  The index.md will can/will be removed once the published site is the “go to” location, as the site is maintained by GitHub Actions, while the index.md requires a PR (like this one!).

Also fixed 213 that I missed.

Fixed - please confirm and let’s get this puppy merged! :-)

JamesKEbert reviewed (approved):
LGTM, let's get it merged!
